modules/custom/d_commerce/modules/d_commerce_products_list/d_commerce_products_list.module

```diff
@@ -2,7 +2,7 @@
 
 /**
  * @file
- * d_commerce_products_list module.
+ * D_commerce_products_list module.
```

Comment:
The machine name should not start in uppercase, maybe "Contains d_commerce_products_list module" would fit better here :-)